apps/routes/lib/repo.ex

```diff
@@ -47,11 +47,13 @@ defmodule Routes.Repo do
     end
   end
 
-  @spec get_shapes(String.t(), 0 | 1, boolean) :: [Shape.t()]
-  def get_shapes(route_id, direction_id, filter_negative_priority? \\ true) do
+  @spec get_shapes(String.t(), Keyword.t(), boolean) :: [Shape.t()]
+  def get_shapes(route_id, opts, filter_negative_priority? \\ true) do
```

Comment:
so the idea is that, now that `direction_id` is optional, we can call `get_shapes` without it to get all shapes in both directions for the route in question when we fetch all the data for this new feature?

Reply:
yes